request_test.go

```diff
@@ -1,20 +1,37 @@
-package hat
+package hat_test
 
 import (
 	"net/http"
+	"net/http/httptest"
 	"net/url"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+
+	"go.coder.com/hat"
+	"go.coder.com/hat/asshat"
 )
 
 func TestURLParams(t *testing.T) {
```

Comment:
totally meaningless change but can we add t.Parallel()?

Reply:
sure